src/main/webapp/WEB-INF/static/js/trialmanager/germplasm/germplasm.js

```diff
@@ -787,6 +787,23 @@
 									return '';
 								}
 							});
+						} else if (columnData.termId === 8254) {
+							// BREEDING_METHOD_ABBR
+							columnsDef.push({
+								targets: columns.length - 1,
+								orderable: false,
+								createdCell: function (td, cellData, rowData, rowIndex, colIndex) {
+									$(td).val("");
+									var value = rowData.properties['8254'].value;
+									if (value) {
+										$(td).append($compile('<a class="gid-link" href="javascript: void(0)" ' +
+											'ng-click="openBreedingMethodModal(\'' + rowData.breedingMethodId + '\')">' + value + '</a>')($scope));
```

Comment:
@cheryleigh it may be possible to use abbreviation here, even if it is out of scope, just search by ID and not over complicate the result of the entries